src/ui/zcl_abapgit_gui_view_repo.clas.abap

```diff
@@ -220,6 +223,11 @@ CLASS zcl_abapgit_gui_view_repo IMPLEMENTATION.
                          iv_act = |{ zif_abapgit_definitions=>c_action-repo_refresh_checksums }?{ lv_key }|
                          iv_opt = lv_crossout ).
 
+    IF mo_repo->get_dot_abapgit( )->get_master_language( ) <> sy-langu.
```

Comment:
According to the blog `cl_abap_syst=>get_logon_language` should be better for this than `sy-langu` ?

Reply:
Fixed